README.md

```diff
@@ -583,6 +583,66 @@ _You can enable the following settings in Xcode by running [this script](resourc
     listingUrgencyBookedShortRowContent(),
   ]
   ```
+  
+* <a id='multi-line-conditions'></a>(<a href='#multi-line-conditions'>link</a>) **Multi-line conditional statements should break after the leading keyword.** Indent each individual statement by [2 spaces](https://github.com/airbnb/swift#spaces-over-tabs). Put the open curly brace on the next line. In `guard` statements, place the `else` keyword on the same line as the open curly brace. [![SwiftFormat: wrapArguments](https://img.shields.io/badge/SwiftFormat-wrapArguments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapArguments) [![SwiftFormat: wrapMultilineStatementBraces](https://img.shields.io/badge/SwiftFormat-wrapMultilineStatementBraces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapMultilineStatementBraces) [![SwiftFormat: elseOnSameLine](https://img.shields.io/badge/SwiftFormat-elseOnSameLine-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#elseonsameline)
```

Comment:
`wrapMultilineStatementBraces` applies to existing rules, like [long-function-declaration](https://github.com/airbnb/swift#long-function-declaration) as well. I'm not sure to what degree we want/expect a 1:1 mapping between Style Guide rules and SwiftFormat rules.

Reply:
> `wrapMultilineStatementBraces` applies to existing rules, like [long-function-declaration](https://github.com/airbnb/swift#long-function-declaration) as well.

👍  Thanks for pointing this out.

> I'm not sure to what degree we want/expect a 1:1 mapping between Style Guide rules and SwiftFormat rules.

I think it's OK to have a many-to-many relationship between rules in our style guide and SwiftFormat rules, and vice versa (e.g. one Airbnb style guide rule could map to one SwiftFormat rule, one Airbnb style guide rule could map to multiple SwiftFormat rules, one SwiftFormat rule could cover multiple Airbnb style guide rules, etc.).

I do think that we should strive to only recommend SwiftFormat rules that are fully covered in our Airbnb style guide. I think it would be a bummer if you adopted our SwiftFormat recommendations to find that they are correcting more than what we cover in our style guide.

Reply:
I'm resolving this thread.